projects/cifuzz-example/project.yaml

```diff
@@ -0,0 +1,8 @@
+language: c++
+sanitizers:
+  - address
+
```

Comment:
did you also want to disable other fuzzing engines?

Reply:
Will it get built by the builders? I thought it won't because it is disabled.

Reply:
The builders shouldn't build it at all. I was thinking about CIFuzz, but sounds like we don't use fuzzing engines other than libFuzzer, right?

Reply:
that's correct.